libs/ballot-interpreter/src/hmpb-rust/interpret.rs

```diff
@@ -625,6 +596,22 @@ pub fn interpret_ballot_card(
     })
 }
 
+pub fn par_map_pair<T, F, R>(left: T, right: T, mapper: F) -> (R, R)
```

Comment:
What does the `par_` prefix here indicate? Parallel? If so, would you be open to calling this `parallel_map_pair`, `parallelized_map_pair`, `map_pair_parallelized`, etc.? Unless `par` is an established Rust convention, of course

Reply:
It's not so much a Rust idiom as the convention used by the [`rayon`](https://crates.io/crates/rayon) crate (e.g. `par_iter`). I don't feel strongly about it, but I am inclined to leave it for this reason.